sklearn/tree/_criterion.pxd

```diff
@@ -40,6 +40,14 @@ cdef class Criterion:
     cdef double weighted_n_left          # Weighted number of samples in the left node
     cdef double weighted_n_right         # Weighted number of samples in the right node
 
+    cdef double* node_sum_total          # An array of the current node value
+    cdef double* node_sum_left           # An array of the left node value
+    cdef double* node_sum_right          # An array of the right node value
```

Comment:
Just one thing though, how are these fields differents from the previous `label_count_*`? You switched to these names because they are shared for both classification and regression?